src/components/Button.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+// eslint-disable-next-line react/prefer-stateless-function
+const Button = ({ color, wide, btnName }) => {
+  const styleColor = ['/', 'X', '-', '+', '='].includes(btnName)
+    ? color[0]
+    : color[1];
+  const widthSize = btnName === '0' ? 345 : wide;
+  return (
+    <div
+      style={{ width: widthSize, backgroundColor: styleColor }}
+      className="calc-btn"
+    >
+      {btnName}
+    </div>
+  );
+};
+
+Button.propTypes = {
+  wide: PropTypes.number.isRequired,
+  color: PropTypes.objectOf(PropTypes.object()).isRequired,
```

Comment:
- [ ] The prop color is supposed to be a string and it does not need to be required, that's it, it will be an optional prop. We can simply define a default value for it, which is going to be orange. We will have something like this:
```javascript
Button.propTypes = {
  wide: PropTypes.boolean.isRequired,
  color: PropTypes.string,
```
And the just bellow it, you can define the default value.
```javascript
Button.defaultProps = {
  color: '#F5913E'    //Default orange color from the asset provided by Microverse
}
```

This way, you will not need to inform the color prop when the button is supposed to have the default color:
```javascript
<Button
  key={btnName} //default value for prop color
  btnName={btnName}
/>
```